bpf/types.h

```diff
@@ -238,7 +218,20 @@ struct filter_value_t {
     u8 filter_drops;
     u8 ip[IP_MAX_LEN];
 } __attribute__((packed));
-// Force emitting struct filter_value_t into the ELF.
-const struct filter_value_t *unused9 __attribute__((unused));
+
+// Force emitting enums/structs into the ELF
+const enum tcp_flags_t *unused0 __attribute__((unused));
```

Comment:
I intentionally added that line after each structure definition as reminder so bpf2go can generate eqv go struct for userspace

Reply:
They're just moved at the end of the file. I can rollback that, I don't really care, but I was finding it easier to choose the next "unusedX" number when having all of them in sight at once

Reply:
not grouping them to gether make it easy to forget but at the cost of looking for the next unused manually

Reply:
ok i'll rollback that

Reply:
addressed via https://github.com/netobserv/netobserv-ebpf-agent/pull/469/commits/01950d5b862e9ce494de90caf87a8846baa4b89d